dags/analyzer_helper/discord/discord_extract_raw_members.py

```diff
@@ -0,0 +1,36 @@
+from analyzer_helper.discord.extract_raw_member_base import ExtractRawMembersBase
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class DiscordExtractRawMembers(ExtractRawMembersBase):
+    def __init__(self, platform_id: str, guild_id: str):
+        """
+        Initialize the class for a specific platform
+        """
+        self.platform_id = platform_id
+        self.guild_id = guild_id
+        self.client = MongoSingleton.get_instance().client
+        self.db = self.client[guild_id]
+        self.collection = self.db["guildmembers"]
+
+    def extract(self, recompute: bool = False) -> list:
+        """
+        Extract members data
+        if recompute = True, then extract the whole members
+        else, start extracting from the latest saved member's `joined_at` date
+
+        Note: if the user id was duplicate, then replace.
+        """
+        if recompute:
+            return list(self.collection.find({}))
+        else:
+            # Fetch the latest joined date
+            latest_member = self.collection.find_one(sort=[("joinedAt", -1)])
```

Comment:
We need to get the latest joined from the `rawmembers` collection under platform id database, to see where to start processing the guildmembers data.
So the flow for recompute = False, would be like
1. Get the latest joined at of the `rawmembers` document collection
2. Extract the documents of `guildmembers` collection under guild_id database after the latest joined at.

Also, please make sure we have just one return expression in the method. It could be saving the data into a variable and then returning that variable at the end of the method.